chapters/data/main.tex

```diff
@@ -604,292 +597,275 @@ \subsection{Pairwise correlation}\label{subsec:diab_correlation}
-seasonal behaviour in a regression model has more to do more with the semantics
-of finding a ``good'' regression model than was intended here but it is an
-important concept nonetheless. If the purpose of this exercise was to accurately
-predict the quantities being plotted rather than just seeing the general trend,
-then a more elaborate model would have been fitted.
+Across each of the models displayed here, there appears to be some seasonality.
+The handling of seasonal behaviour in a regression model has more to do more
+with the semantics of finding a ``good'' regression model than was intended here
+but it is an important concept nonetheless. If the purpose of this exercise was
+to accurately predict the quantities being plotted rather than just seeing the
+general trend, then a more elaborate model should have been fitted.
+
+
+\section{Chapter summary}\label{sec:summary}
+
+This chapter provided a descriptive analysis of an administrative dataset
+provided by the Cwm Taf Morgannwg UHB. This dataset describes the episodes of
+all patients being treated at two of their hospitals over a five year period.
+The scope of this exploration of the dataset was thwarted by the span and
+diversity of the episodes therein. In fact, there was little insight to be
+gained beyond its conforming to some loosely anecdotal concept of a
+administrative healthcare dataset.
+
+Following this initial analysis, a similar methodology was employed on a slice
+of the dataset. As an act of familiarity, the diabetic population was used. The
+hope of that analysis was that taking a slice according to some characteristic
+would result in a more homogeneous dataset, which was confirmed, although only
+marginally. Other than this, there were very few actionable insights to be
+gained from this sort of exploratory analysis.
+
+So what was there to gain by looking at the diabetic population? The lack of
+novel insights may have been expected since the decision to look at diabetic
+patients was effectively arbitrary. Moreover, the decision framework was not
+descriptive enough to indicate that any particular kind of patient was being
+investigated other than that they must exhibit this one condition. So, in that
+way, there was little to gain. However, as has been noted throughout this
+chapter, taking a subset of the population allows for some comparison with its
+complement, as well as the general population.
+
+The next chapter considers another administrative dataset comprised of patients
+presenting another chronic condition. The work in that chapter incorporates some
+of the techniques introduced in the previous chapters, but it also builds on the
+surface-level analyses employed here. The most important of these analyses is
+the slicing of a dataset to identify more homogeneous parts, and then comparing
+those. The process by which this is done incorporates more attributes of the
+data than the condition alone, or age, as is commonly done. Ultimately, the
```

Comment:
Is it worth here, connecting this chapter to the other chapters? Perhaps just indicating that in the next chapter we will use techniques discussed in the previous chapters?

Reply:
Isn't that covered by lines 861-864?

Reply:
I did somehow miss that (🤦 ) but I think it could be even more explicit. 

```
The work in that chapter will rely on / use ... presented in chapter ...
```

Reply:
Fixed by 89c7a11 (maybe?)